.github/workflows/ci.yaml

```diff
@@ -46,6 +46,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: Set up python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install automation scripts dependencies
+      run: pip install click
```

Comment:
If there is now an `automation` directory, and scripts there have their own independent requirements, maybe we should have a `requirements.txt` in that dir and install it here with `-r` instead of adding another requirement here every time we'll need too.

Reply:
definitely